app/livechat/client/views/sideNav/livechatSideNavItems.js

```diff
@@ -0,0 +1,24 @@
+export const sidebarItems = new ReactiveVar([]);
+export const addSidebarItem = (title, slug) => {
+	sidebarItems.set([
+		...sidebarItems.get(),
+		{
+			title,
+			slug,
+		}
+	]);
+};
+
+addSidebarItem("Current_Chats", "livechat-current-chats");
```

Comment:
Let's use `single quotes` here according to the eslint rules.
```suggestion
addSidebarItem('Current_Chats', 'livechat-current-chats');
```